redisinsight/api/package.json

```diff
@@ -10,9 +10,10 @@
   },
   "scripts": {
     "build:defaults:commands": "ts-node ./scripts/default-commands.ts",
-    "build:defaults:enablement-area": "ts-node ./scripts/default-enablement-area.ts",
+    "build:defaults:guides": "ts-node ./scripts/default-guides.ts",
+    "build:defaults:tutorials": "ts-node ./scripts/default-tutorials.ts",
     "build:defaults:content": "ts-node ./scripts/default-content.ts",
-    "build:defaults": "yarn build:defaults:enablement-area && yarn build:defaults:commands yarn build:defaults:content",
+    "build:defaults": "yarn build:defaults:guides && yarn build:defaults:commands && yarn build:defaults:content",
     "prebuild": "rimraf dist",
```

Comment:
Did you test it locally? I assume that default tutorials will not work because I don't see `build:defaults:tutorials` here

Reply:
I didn't include the tutorials because there are no files in the repository, so the script will fail with an error